src/main/java/com/qaprosoft/carina/demo/phpTravels/mobile/common/FacebookPageBase.java

```diff
@@ -0,0 +1,10 @@
+package com.qaprosoft.carina.demo.phpTravels.mobile.common;
+
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+
+public class FacebookPageBase extends AbstractPage {
```

Comment:
for mobile classes please use this naming strategy:
`FacebookHomeScreenBase` for base class and `FacebookHomeScreen` for children classes
`SettingsNativeAppScreenBase` for base class and `SettingsNativeAppScreen` for children classes